add step for Select-AzureRmSubscription

christothes opened the pull request:
Added the step to set the current subscription context in the case that the user has many (which I did). The deploy command won't find a pre-existing ResourceGroup if it exists in a different subscription that the context set by Login-AzureRmAccount.

azurecla commented:
Hi __@lakasa-msft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

christothes committed:
add step for Select-AzureRmSubscription

Added the step to set the current subscription context in the case that the user has many (which I did). The deploy command won't find a pre-existing ResourceGroup if it exists in a different subscription that the context set by Login-AzureRmAccount.
